test/e2e/data/autoscaler/autoscaler-to-workload-management.yaml

```diff
@@ -0,0 +1,58 @@
+# This yaml creates a service account to be used by the autoscaler running on a workload cluster
```

Comment:
I'm not sure if I'm missing something, but it looks like we need the exact same that we achieve with `getAuthenticationToken` in kcp_remediations.go. The only difference seems to be the permissions (Role.rules).

Can we consider making `getAuthenticationToken` slightly more generic and then just reuse it in the autoscaler test? (I"m also fine with just copying it and adjusting it)

Note: I think we have to make "dockermachinetemplates" a parameter of the autoscaler test. But that is better than describing how infra provider have to copy&paste and adjust the entire autoscaler-to-workload-management.yaml.

Reply:
Note: If this test currently does not work on linux, implemeting this change might solve the problem (in combination with getServerAddr)